src/builder/Accounts.sol

```diff
@@ -37,6 +38,25 @@ library Accounts {
         uint256 balance;
     }
 
+    struct CometPosition {
+        address comet;
+        CometBasePosition basePosition;
+        CometCollateralPosition[] collaterals;
```

Comment:
nit: `collateralPositions`

Reply:
I went with `collaterals` since that's what the existing backend calls that field (I failed to be consistent, however, and renamed `base` to `basePosition` 🤦 ).

Does it matter whether the field names are the same as the backend?

```
"cometPositions" => [
    %{
      "comet" => "0x2943ac1216979ad8db76d9147f64e61adc126e96",
      "base" => %{
        "asset" => "0x2d5ee574e710219a521449679a4a7f2b43f046ad",
        "accounts" => [
          "0xf6F148cFb92dD3baF399B0eCD1a8561b7f16487D",
          "0xE276Bc378A527A8792B353cdCA5b5E53263DfB9e"
        ],
        "borrowed" => [0, 0],
        "supplied" => [0, 0],]
      },
      "collaterals" => [
        %{
          "asset" => "0xb9fa8f5ec3da13b508f462243ad0555b46e028df",
          "accounts" => [
            "0xf6F148cFb92dD3baF399B0eCD1a8561b7f16487D",
            "0xE276Bc378A527A8792B353cdCA5b5E53263DfB9e"
          ],
          "balances" => [0, 0]
        },
      ]
    }
 ]
```

Reply:
updated to `collateralPositions`